homeassistant/components/switcher_kis/cover.py

```diff
@@ -117,16 +134,20 @@ async def _async_call_api(self, api: str, *args: Any) -> None:
 
     async def async_close_cover(self, **kwargs: Any) -> None:
         """Close cover."""
-        await self._async_call_api(API_SET_POSITON, 0)
+        index = get_circuit_number(self._cover_id)
+        await self._async_call_api(API_SET_POSITON, 0, index)
```

Comment:
The implementation of the `get_circuit_number` here does not make sense, however since it is not used in this PR (it will simply return `0`) please remove it and remove the `index` parameter, you can add it in a future PR when you add support for dual cover.

Reply:
Removed `get_circuit_number ` and `index`